app/inbound/total/process-total.js

```diff
@@ -0,0 +1,29 @@
+const db = require('../../data')
+
+const getTotalByCalculationReference = require('./get-total-by-calculation-reference')
+const savePlaceholderOrganisation = require('./save-placeholder-organisation')
+const saveTotal = require('./save-total')
+const saveActions = require('./save-actions')
+const validateTotal = require('./validate-total')
+
+const processTotal = async (total) => {
+  const transaction = await db.sequelize.transaction()
+  try {
+    const existingTotal = await getTotalByCalculationReference(total.calculationReference, transaction)
```

Comment:
getTotalByCalculationReference should be getTotalByCalculationId

Reply:
the function is called getTotalByCalculationReference as it's in calculationReference format at that point and is only referencing calculationId which is only later converted in the saveTotal function.

At that point, calculationId still doesn't exist.

Reply:
renamed Reference to Id as suggested